src/MessagingAvatar/MessagingAvatar.tsx

```diff
@@ -28,27 +28,33 @@ export const MessagingAvatar: React.FC<Props> = ({ className, text, color }: Pro
       // If all else fails, we can use text as the seed to determine color.
       style={{
         backgroundColor:
-          "color" in color
-            ? color.color
-            : `hsl(${_determineAvatarHue(color.seed || text)}, 80%, 85%)`,
+          "color" in color ? color.color : `#${_determineAvatarColor(color.seed || text)}`,
       }}
     >
       <div className={cssClasses.TEXT}>{text}</div>
     </div>
   );
 };
 
-// Helper function: Uses a string to determine the hue value for an HSL
-//  (Hue, Saturation, Lightness) color, to be used in an inline style.
-//  Unlikely to be the final method, waiting for direction from Design.
-function _determineAvatarHue(determiningString: string): number {
+// Helper function: Uses a string as the seed to fairly randomly determine
+//  the hex color for this avatar.
+function _determineAvatarColor(determiningString: string): string {
+  const colorPool = [
+    "C9F2F1",
+    "FDB9C0",
+    "FAE5B8",
+    "D9BDFD",
+    "B4EAD8",
+    "BFCEFF",
+    "FFBADE",
+    "FDBD81",
+  ];
+
   let hash = 0;
   for (let i = 0; i < determiningString.length; i++) {
     hash = determiningString.charCodeAt(i) + ((hash << 5) - hash);
     hash = hash & hash; // Converts to a 32bit int
   }
 
-  // Hue values for HSL color function are between 0 and 359,
-  //  for degrees on the color wheel.
-  return hash % 360;
+  return colorPool[Math.abs(hash % colorPool.length)];
```

Comment:
Instead of modulo 360 for HSL hue value, modulo `colorPool.length`. As far as I can tell, this works pretty well with mongo IDs, names, whatever!

Reply:
Nice!